### ISSUE TITLE
Swapped ninserts/ndeletes in bde_control.upload_stats

### ISSUE THREAD

**Issue by strk** [linz/linz-bde-uploader#165]
The number of inserts and number of deletes in `bde_control.update_stats` table are swapped.
This is due to `bde_control.bde_ApplyLevel5Update` taking the results from `table_version._ver_apply_changes` in the wrong order:
```  
        SELECT * FROM table_version._ver_apply_changes(
            v_bdetable, v_tmptable, '_tmp_inc_actions', v_key_column
        )
        INTO v_ndel, v_nins, v_nupd;
```

The  `_ver_apply_changes` function in `table_version` is defined to return inserts first:
```
CREATE OR REPLACE FUNCTION _ver_apply_changes(
    p_original_table REGCLASS,
    p_new_table REGCLASS,
    p_diff_table NAME,
    p_key_column NAME,
    OUT number_inserts BIGINT,
    OUT number_deletes BIGINT,
    OUT number_updates BIGINT
)   
```

This hasn't changed since the beginning of `table_version`.
The code in linz-bde-uploader was probably copied from the code in table_version.ver_apply_changed (w/out the underscore) which used to be also bogus until version 1.3.0: https://github.com/linz/postgresql-tableversion/issues/39

**Comment by strk** [comment]
I think the fix for this bug should also include a fix of the _data_ in the database, swapping values if coming from a version before the fix. This may mean we'd better put the fix in 2.5.0 instead of 2.4.1, for easier version detection.

\cc @palmerj

**Comment by imincik** [comment]
This task is still relevant.
